website/scripts/sync-dandelion-apps.js

```diff
@@ -0,0 +1,28 @@
+const { syncPages } = require('./sync-util')
+
+const GIT_REF = 'master'
```

Comment:
This seems like it will only work for one repo?

Reply:
Works for an indefinite amount of repos